src/App.js

```diff
@@ -1,43 +1,26 @@
-import "./App.css";
-import logo from "./logo.png";
+import {Routes, Route} from "react-router-dom"
+import {Homepage} from "./pages/Homepage/Homepage"
+import { ProductListing } from "./pages/ProductListing/ProductListing"
+import { WishlistPage } from "./pages/WishlistPage/WishlistPage"
+import { CartPage } from "./pages/CartPage/CartPage"
+import { LoginPage } from "./pages/LoginPage/LoginPage"
```

Comment:
You can make in index file in pages folder and export all the components from there and import it in here from index.js only

Reply:
I've just  made a pages.js file and from there i am importing . Thanks for suggestion